dev/io.openliberty.jaxws.security_fat.ssl/.classpath

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="fat/src"/>
+	<classpathentry kind="src" path="test-applications/WsdlLocationHttpsServer/src"/>
+	<classpathentry kind="src" path="test-applications/WsdlLocationHttpsClient/src"/>
```

Comment:
These class path entries for the test applications look wrong, as they don't seem to match what's actually being used by the tests here. I can't find any reference to the some of these apps in the test classes. You should remove the entries to the tests apps that aren't actually being used in this project.